docs/en/03.venus-worker-config.md

```diff
@@ -707,6 +707,74 @@ concurrent = 13
 envs = { FIL_PROOFS_USE_MULTICORE_SDR = "1" }
 ````
 
+#### `cgroup.cpuset` Configuration Notes
+* The number of CPU cores in `cgroup.cpuset` must be an integer multiple of the corresponding number of CPU caches
+
+According to [rust-fil-proofs](https://github.com/filecoin-project/rust-fil-proofs), when `multicore sdr` is enabled, the number of CPU cores and number of corresponding CPU caches must be an integer multiple relationship. If the external processor is using rust-fil-proofs or rust-fil-proofs based variants, this rule must be followed, otherwise it can be ignored. The default processor of venus-worker is based on rust-fil-proofs.
```

Comment:
```suggestion
According to [rust-fil-proofs](https://github.com/filecoin-project/rust-fil-proofs), when `multicore sdr` is enabled, the number of CPU cores and number of corresponding cores on single DIE must be an integer multiple relationship. If the external processor is using rust-fil-proofs or rust-fil-proofs based variants, this rule must be followed, otherwise it can be ignored. The default processor of venus-worker is based on rust-fil-proofs.
```